cmd/operator-sdk/new/cmd.go

```diff
@@ -303,8 +241,8 @@ func doAnsibleScaffold() error {
 }
 
 func verifyFlags() error {
-	if operatorType != projutil.OperatorTypeAnsible && operatorType != projutil.OperatorTypeHelm {
-		return fmt.Errorf("value of --type can only be `ansible`, or `helm`: %v",
+	if operatorType != projutil.OperatorTypeAnsible {
+		return fmt.Errorf("value of --type can only be `ansible`: %v",
 			projutil.ErrUnknownOperatorType{Type: operatorType})
 	}
 	if operatorType != projutil.OperatorTypeAnsible && generatePlaybook {
```

Comment:
Nit: this `if` block can be removed, since it will _always_ be an ansible project at this point.